lib/mix/tasks/playlist.ex

```diff
@@ -1,7 +1,8 @@
-defmodule App do
-  use Application
+defmodule Mix.Tasks.Playlist do
+  use Mix.Task
```

Comment:
I don't think you want to remove you application, which defines the supervision tree and belongs there. You should probably just remove the last call.

The mix task is a nice way to run your one-off thing 👍
Another idea could be to run `mix run -e Core.create_playlist` directly ([doc](https://hexdocs.pm/mix/Mix.Tasks.Run.html)).

Reply:
I see, updated. Thanks!